### PR TITLE
adding comments for create scene without csb file

### PR DESCRIPTION
**Pull request by GuoLunHao**
@xiaofeng11 @pandamicro @natural-law

### CODE REVIEW

File: plugins/plugin_generate/bin-templates/js-template-runtime/src/app.js [hunk 1]
```diff
@@ -15,6 +15,27 @@ var HelloWorldLayer = cc.Layer.extend({
         var mainscene = ccs.load(res.MainScene_json);
```
Comment (pandamicro) [inline plugins/plugin_generate/bin-templates/js-template-runtime/src/app.js:15]: There is no res.MainScene_json by default in js_template_runtime, how is it added ?

**Comment by GuoLunHao** [comment]
this bin-templates folder is used by command 'cocos gen-templates' to generate 'cpp-templates-binary','js-templates-binary' and 'lua-templates-binary'. 'cocos new -l lua -t binary luaProject'  can create a Cocos Project with prebuilt library. these comments is added for showing users how to use cocos prebuilt library without cocostudio project.

File: plugins/plugin_generate/bin-templates/js-template-runtime/src/app.js (hunk 1 above)
Comment (GuoLunHao) [inline plugins/plugin_generate/bin-templates/js-template-runtime/src/app.js:15]: res.MainScene_json is defined in 'plugins / plugin_generate / bin-templates / js-template-runtime / src / resource.js ' , and the resouce file is in the path' plugins / plugin_generate / bin-templates / js-template-runtime / res-landscape / res / ' or the path'plugins / plugin_generate / bin-templates / js-template-runtime / res-res-portrait  / res / '

**Comment by natural-law** [comment]
@pandamicro 
It's OK for merge.

**Pull request merged**